kalite/distributed/static/js/distributed/color_palette/color_palette.js

```diff
@@ -0,0 +1,127 @@
+var $ = require("../base/jQuery");
+require("browsernizr/test/inputtypes");
+var Modernizr = require("browsernizr");
+var colorPaletteTemplate = require("./hbtemplates/color-palette.handlebars");
+var BaseView = require("../base/baseview");
+
+var ColorPaletteView = BaseView.extend({
+    template: colorPaletteTemplate,
+    tagName: "style", 
+
+    initialize: function() {
+        this.render();
+    },
+
+    render: function(){
+        this.$el.html(this.template({
+            k_bg_clr: $("#my_background").val(),
+            k_acn_clr: $("#my_accent").val(),
+            k_act_clr: $("#my_action").val(),
+            k_hln_clr: $("#my_headline").val(),
+            k_byt_clr: $("#my_bodytext").val(),
+        }));
+    }
+});
+
+$(function(){
```

Comment:
Could some of this logic be encapsulated in another Backbone View? We can then just delegate the initialization of that view to the ```$()``` wrapper.

Would also be good to put the ```$()``` in a separate file, so we can expose the colour picker code as a pluggable module, separate from its in page instantiation.